requirements/edx-platform-constraints.txt

```diff
@@ -41,18 +41,18 @@ django-babel-underscore==0.5.2
 django-babel==0.6.2       # via django-babel-underscore
 git+https://github.com/edx/django-celery.git@756cb57aad765cb2b0d37372c1855b8f5f37e6b0#egg=django-celery==3.2.1+edx.2
 django-classy-tags==1.0.0  # via django-sekizai
-django-config-models==1.0.4
+django-config-models==2.0.0
 django-cors-headers==2.5.3
 django-countries==5.5
 django-crum==0.7.5
 django-fernet-fields==0.6
 django-filter==2.2.0
 django-ipware==2.1.0
 django-js-asset==1.2.2    # via django-mptt
-django-method-override==0.2.0
+git+https://gitlab.com/Ayub-khan/django-method-override.git@5270af321be2e576d8e8b3c4191711a19975c356#egg=django-method-override==1.0.4
```

Comment:
is this coming from the platform ?

Reply:
Yes, this file is generated automatically by running `make upgrade` and it fetches requirement constraints from edx-platform.